pyproject.toml

```diff
@@ -42,7 +42,6 @@ pytest-cov = "^2.10.1"
 pytest-icdiff = "^0.5"
 
 [tool.poetry.scripts]
-datagateway-api = "datagateway_api.src.main:run_api"
```

Comment:
What does this removal do? Based off the commit messages ("Add codecov dependency"/"Remove codecov as a dependency") I don't know why it's been changed, but equally I'm sure there could be a good reason for it to be gone.

Reply:
This was a (non-working) entrypoint to the API that I was experimenting with when I added Poetry to this repo. It never worked and I forgot to remove it, so I removed it while I was looking through the poetry files making sure I removed codecov. But agreed, confusing, I should've mentioned it as a bullet point within the commit message